src/test/resources/FeatureFilesAndreeaC/tema1.feature

```diff
@@ -0,0 +1,31 @@
+Feature: Testing login page
+
+  Scenario Outline: Verify login
+    Given I open login page
+    When I enter <user> and <pass>
```

Comment:
aici iti recomand sa folosesti asa: I enter username <user> and password <pass> ca sa se inteleaga ce sunt parametrii aia

Reply:
Done